<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>558</width>
                <height>484</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QPushButton" name="btn1">
                <property name="geometry">
                    <rect>
                        <x>440</x>
                        <y>50</y>
                        <width>93</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>함초롬바탕</family>
                        <pointsize>12</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>삭제</string>
                </property>
            </widget>
            <widget class="QPushButton" name="btn2">
                <property name="geometry">
                    <rect>
                        <x>440</x>
                        <y>10</y>
                        <width>93</width>
                        <height>28</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>함초롬바탕</family>
                        <pointsize>12</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>추가</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="tb_name">
                <property name="geometry">
                    <rect>
                        <x>140</x>
                        <y>10</y>
                        <width>251</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>함초롬바탕</family>
                        <pointsize>12</pointsize>
                    </font>
                </property>
            </widget>
            <widget class="QLabel" name="label">
                <property name="geometry">
                    <rect>
                        <x>30</x>
                        <y>10</y>
                        <width>64</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>함초롬바탕</family>
                        <pointsize>12</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>이름</string>
                </property>
            </widget>
            <widget class="QLabel" name="label_2">
                <property name="geometry">
                    <rect>
                        <x>30</x>
                        <y>50</y>
                        <width>111</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>함초롬바탕</family>
                        <pointsize>12</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>생년월일</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="tb_birth">
                <property name="geometry">
                    <rect>
                        <x>140</x>
                        <y>50</y>
                        <width>251</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>함초롬바탕</family>
                        <pointsize>12</pointsize>
                    </font>
                </property>
            </widget>
            <widget class="QLabel" name="label_3">
                <property name="geometry">
                    <rect>
                        <x>30</x>
                        <y>90</y>
                        <width>91</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>함초롬바탕</family>
                        <pointsize>12</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>비밀번호</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="tb_pw">
                <property name="geometry">
                    <rect>
                        <x>140</x>
                        <y>90</y>
                        <width>251</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>함초롬바탕</family>
                        <pointsize>12</pointsize>
                    </font>
                </property>
            </widget>
            <widget class="QPushButton" name="btn1_2">
                <property name="geometry">
                    <rect>
                        <x>440</x>
                        <y>90</y>
                        <width>91</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>함초롬바탕</family>
                        <pointsize>12</pointsize>
                    </font>
                </property>
                <property name="text">
                    <string>실행</string>
                </property>
            </widget>
            <widget class="QTableWidget" name="table">
                <property name="geometry">
                    <rect>
                        <x>20</x>
                        <y>140</y>
                        <width>511</width>
                        <height>321</height>
                    </rect>
                </property>
            </widget>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
